pkg/cmd/init.go

```diff
@@ -14,6 +14,10 @@ func NewInitCmd(cf *genericclioptions.ConfigFlags) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "init",
 		Short: "Create custom resource definitions used by starboard",
+		Long: `Create custom resource definitions and ConfigMap used by starboard
+in the starboard namespace. This contains the default configuration
+parameters. However this can be modified to change the behaviour of
+the scanner.`,
```

Comment:
For the sake of completeness I'd also mention that we create:
- starboard service account
- starboard cluster role and cluster role binding (subject to change in the future but currently we do that)

Do you think we can also mention the cleanup command, which does the revers of init?

Reply:
@danielpacak Thanks for the feedback. Based on that I have had another go at rewriting the long message.

Note: Any feedback is appreciated.